tests/foreman/destructive/test_contenthost.py

```diff
@@ -45,6 +45,7 @@ def test_content_access_after_stopped_foreman(target_sat, rhel7_contenthost):
     :parametrized: yes
     """
     org = target_sat.api.Organization().create()
+    org.sca_disable()
```

Comment:
I think disabling sca might not be required for 6.12.z, right?
```suggestion
```

Reply:
Correct.